piet/src/text.rs

```diff
@@ -78,7 +78,7 @@ pub trait TextLayout: Clone {
 
     /// Used for changing the width of a text layout. Given a width, returns a [`TextLayout`]
     /// struct with recalculated lines and line metrics.
```

Comment:
```suggestion
    /// Change the width of this `TextLayout`.
    ///
    /// This may be an `f64`, or `None` if this layout is not constrained;
    /// `None` is equivalent to `std::f64::INFINITY`.
```